_build/eleventy.js

```diff
@@ -4,7 +4,7 @@ import markdownItDeflist from "markdown-it-deflist";
 import pluginTOC from "eleventy-plugin-toc";
 import * as filters from "./filters.js";
 
-import components from "prismjs/src/components.json" with { type: "json" };
+import components from "../node_modules/prismjs/src/components.json" with { type: "json" };
```

Comment:
Why does this work? Should it resolve to the same thing as `prismjs`?

Reply:
As far as I understand, when importing directly from `prismjs`, the export rules from `package.json` are in play. We don't expose the `src` folder there (which I think is right if we don't plan to publish Prism with the source files), so to access files from there directly, we need to import from `node_modules`. I see it this way.